@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -651,6 +651,18 @@ export class TiltBasedClipPlanesEvaluator extends AltitudeBasedClipPlanesEvaluat
         // Near plance calculus is pretty straightforward and does not depent on camera tilt:
         const cameraAltitude = this.getCameraAltitude(camera, projection);
         viewRanges.near = cameraAltitude - this.maxElevation;
+        if (camera instanceof THREE.PerspectiveCamera) {
```

Comment:
Tilting is not only restricted to PerspectiveCamera

Reply:
Yes but for Perspective one we need to make little bit more calculus. Either way I have already found some nice solution to simplify the code in this section, but I would not like to extend this PR with another commit. I would be more then happy to work on this a little more but with some new PR (ticket), cause its already difficult to maintain this one (rebase/merge cycle).

Reply:
New solution has been uploaded as last commit, but of course still Perspective and Orthographics camera projections are differentiated. Anyway the code for TiltBased... is much more simpler and I think clearer, the graphs presenting math used are on the solutioning page now... I will try to write also few explanatory sentences.